actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -36,22 +36,24 @@ module Redirecting
     # If you are using XHR requests other than GET or POST and redirecting after the
     # request then some browsers will follow the redirect using the original request
     # method. This may lead to undesirable behavior such as a double DELETE. To work
-    # around this  you can return a <tt>303 See Other</tt> status code which will be
+    # around this you can return a <tt>303 See Other</tt> status code which will be
     # followed using a GET request.
     #
     #   redirect_to posts_url, status: :see_other
     #   redirect_to action: 'index', status: 303
     #
-    # It is also possible to assign a flash message as part of the redirection. There are two special accessors for the commonly used flash names
+    # It is also possible to assign a flash message as part of the redirection.
+    # There are two special accessors for the commonly used flash names
     # +alert+ and +notice+ as well as a general purpose +flash+ bucket.
     #
     #   redirect_to post_url(@post), alert: "Watch it, mister!"
     #   redirect_to post_url(@post), status: :found, notice: "Pay attention to the road"
     #   redirect_to post_url(@post), status: 301, flash: { updated_post_id: @post.id }
     #   redirect_to({ action: 'atom' }, alert: "Something serious happened")
     #
-    # Statements after +redirect_to+ in our controller get executed, so +redirect_to+ doesn't stop the execution of the function.
-    # To terminate the execution of the function immediately after the +redirect_to+, use return.
+    # Statements after +redirect_to+ in our controller get executed, so +redirect_to+
+    # doesn't stop the execution of the function. To terminate the execution of the
+    # function immediately after the +redirect_to+, use return.
```

Comment:
Lets revert the wraps to 80 lines. It is cosmetic change and goes on along when there is a related change on the concerned lines.